src/components/Form/Buttons/styles/BasicButton.style.tsx

```diff
@@ -24,6 +24,14 @@ export const useStyles = makeStyles(
       '&:hover': {
         backgroundColor: '#ffaa00',
       },
+      [theme.breakpoints.down('sm')]: {
+        flex: '0 1 auto',
+        padding: theme.spacing(2, 3.1),
+        '& .MuiButton-label': {
+          fontWeight: 600,
```

Comment:
but why different `font-weight` for BasicButton and CancelButton? 
600 is not wide-spread value - if it's absent in current font, then 700 will be applied([MDN: font-weight#Fallback Weights](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight?retiredLocale=pl#fallback_weights))

does it look consistently across all the supported browsers?

maybe we better stick to "bold"(700) instead?

Reply:
We have the same font-weight for Basic and Cancel, I can try make bold. But, i thought on mobile devices it will not look pretty. Maybe should try normal font-weight?

Reply:
Ok, i checked bold looks normally, will make bold for all devices

Reply:
deal